provision-gcc-VM.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+set -eux
+
+usage() {
+  cat <<EOF
+  usage: $0 -a VM_IP_ADDRESS -p VM_ROOT_PASSWORD
+
+  This script installs the required software and configures the VM,
+  described by its VM_IP_ADDRESS passed as argument, for installing
+  omnibus manually omnibus gem and be able to build codethink-toolchain
+  (https://github.com/CodethinkLabs/omnibus-codethink-toolchain).
+
+  OPTIONS:
+    -h Show this message
+    -a IP address of the VM which is going to be configured.
+    -p root password of the VM which is going to be configured.
+
+  EXAMPLE:
+    $0 -a 192.168.256.256 -p insecure
+EOF
+}
+
+configure_vm_for_gcc() {
+  pushd "$script_dir"
+  ansible-playbook -i hosts user-creation.yml \
+	           -e "ansible_user=root ansible_ssh_pass=$vm_root_password"
+  ansible-playbook -i hosts gcc-vm.yml \
+	           -e "ansible_user=rpm_omnibus ansible_ssh_pass=1ns3cur3"\
```

Comment:
Is there a need to use this `rpm_omnibus` when calling ansible? you can set `become: yes` and `become_user: rpm_omnibus` in this playbook to make sure all tasks are run by this user. This way you avoid hardcoding a password here.

Reply:
I haven't test it, but I'm not sure you can become an user just created in the system. I will keep it in mind for future patches, I won't change this in this series/PR.